README.md

```diff
@@ -43,8 +43,8 @@ Other Commands
 | `yarn docker:stop` | Stops the backend and frontend docker containers |
 | `yarn docker:test` | Runs tests for the frontend and backend in docker containers |
 | `yarn docker:lint` | Runs the linter for the frontend and backend in docker containers |
-| `yarn docker:db:migrate` | Run migrations in docker containers |
-| `yarn docker:db:migrate:undo` | Undo migrations in docker containers |
+| `yarn docker:db:migrate` | Run migrations in docker containers | `yarn db:migrate` | Locally |
+| `yarn docker:db:migrate:undo` | Undo migrations in docker containers | `yarn db:migrate:undo` | Locally |
```

Comment:
When I preview this on github I don't see the `yarn db:migrate`... commands. Maybe we need to redo the table with three columns, `host commands` (not a great name, open to other ideas) `docker commands` and `description`?

Reply:
Good catch.